shared/chat/inbox/container/index.js

```diff
@@ -66,21 +78,19 @@ const mapDispatchToProps = (dispatch, {navigateAppend}) => ({
 
 // This merge props is not spreading on purpose so we never have any random props that might mutate and force a re-render
 const mergeProps = (stateProps, dispatchProps, ownProps) => {
-  const {allowShowFloatingButton, rows, smallTeamsExpanded} = stateProps.filter
-    ? filteredRowData(stateProps._metaMap, stateProps.filter, stateProps._username)
-    : normalRowData(stateProps._metaMap, stateProps._smallTeamsExpanded)
   return {
-    _isLoading: stateProps.isLoading,
```

Comment:
isLoading changes all the time and we don't actually care if its ever false, so encode as canRefreshOnMount doesn't flip back and forth